css/style.css

```diff
@@ -38,6 +38,7 @@ input[type=checkbox].waveform{
 
 /* housing */
 label.waveform {
+     pointer-events: none;
```

Comment:
This element is clickable right? Accessibility features sometimes use cursor state to indicate interactivity; might be good to keep pointer events active on this element.

Reply:
@emcniece this element is not clickable. This is basically a hack to pass the clicking functionality to another element so that I could keep the toggle switch look (but give the functionality to the backend service)

Reply:
Oh I see, clever! I still think there may be a case for making the visible element clickable and keeping pointer-events from an accessibility standpoint, but I'd have to do some reading. Making that happen probably means a small refactor, might be better suited for a separate issue.